src/main/web_src/src/components/fagsystem/inntektstub/form/partials/inntektsinformasjonForm.tsx

```diff
@@ -18,14 +19,22 @@ export default ({ path, formikBag }: InntektsinformasjonForm) => {
 
```

Comment:
Oppdaget ikke dette forrige gang, men nå blir date satt til null hver gang formet blir rendra. Dvs. hvis man velger en dato, og går videre for å så gå tilbake igjen, vises ikke den valgte datoen i inputfeltet. En løsning kunne kanskje være at man i useState sjekker om sisteAarManed er satt, i så fall får date denne verdien, hvis ikke blir den satt til null? 🤔

Reply:
Denne var litt vrien å få til faktisk 😅  Ordna seg etterhvert, men ble kanskje ikke så pent 🤪